app/assets/templates/widget_time_options_template.html

```diff
@@ -26,7 +26,14 @@
             <i class="icon-fb"></i>
           </button>
         </span>
-        <input type="text" class="form-control" datetime-picker datetime="endTime" data-format="yyyy-MM-dd" placeholder="Until">
+        <span class="until_container">
+          <input type="text" class="form-control" datetime-picker datetime="endTime" data-format="yyyy-MM-dd" placeholder="Until">
+          <span class="until_clear"
+            ng-click="clearEndTime()"
+            tooltip-append-to-body="true" tooltip="Clear">
+            <i class="glyphicon glyphicon-remove-circle"></i>
```

Comment:
I think `glyphicon-remove` (without circle) would fit better into the adjacent visual language we're using.

Also, give the glyphicon some (6px?) padding/margin to the right so that it has equal distance from the top/bottom of the input field as from the right.